model/project_aliases.go

```diff
@@ -200,19 +200,35 @@ func FindAliasInProjectOrRepo(projectID, alias string) ([]ProjectAlias, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "error checking for existing aliases")
 	}
-	// If nothing is defined in the DB, check the project config
-	// Unless the alias defined is a patch alias and there are patch aliases
+	// If nothing is defined in the DB, check the project config,
+	// unless the alias defined is a patch alias and there are patch aliases
 	// defined on the project page.
 	if len(aliases) > 0 {
 		return aliases, nil
 	}
 	shouldExit := false
 	if IsPatchAlias(alias) {
-		numPatchAliases, err := countPatchAliasesForProjectRef(projectID)
+		projectRefPatchAliases, err := countPatchAliases(projectID)
```

Comment:
nit: should still use num or count or something in the variable name since this is just an integer and not the actual aliases.